modules/web/pom.xml

```diff
@@ -1,49 +1,42 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-  <modelVersion>4.0.0</modelVersion>
-  <parent>
-    <groupId>com.devonfw.java.dev</groupId>
-    <artifactId>devon4j-modules</artifactId>
-    <version>dev-SNAPSHOT</version>
-  </parent>
-  <groupId>com.devonfw.java.modules</groupId>
-  <artifactId>devon4j-web</artifactId>
-  <version>${devon4j.version}</version>
-  <packaging>jar</packaging>
-  <name>${project.artifactId}</name>
-  <description>Module for reusable web and servlet related code of the Open Application Standard Platform for Java (devon4j).</description>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>com.devonfw.java.dev</groupId>
```

Comment:
something is wrong with your formatter / indendation.
However, this is a nice-to-have that should not prevent the PR being merged.
Anyhow, please check your IDE settings and fix this for further work to make diffs more readable.